src/components/App.js

```diff
@@ -0,0 +1,70 @@
+//import other components to render
+import Header from '../components/Header.js';
+import Component from './Component.js';
+import AddItem from '../components/AddItem.js';
+//todoData is our data array, not a component
+import todoData from '../../data/todo-data.js';
+import TodoList from './TodoList.js';
+
+//THIS FILE RENDERS OUR WEBSITE DYNAMICALLY
```

Comment:
The concepts are very new, I understand the need to take notes. This does make the PR crowded and hard to read. In the future we will be a lot more critical of comments as well as commenting out any code.

Reply:
Thanks for letting me know, I'll be better about cleaning up code before pushing to origin :)